suite-native/module-authorize-device/src/hooks/useOnDeviceReadyNavigation.ts

```diff
@@ -56,6 +56,12 @@ export const useOnDeviceReadyNavigation = () => {
             (isDeviceReadyToUseAndAuthorized && isTimeoutFinished) ||
             (deviceEnabledDiscoveryNetworkSymbols.length === 0 && isCoinEnablingInitFinished)
         ) {
+            if (navigation.canGoBack()) {
```

Comment:
Not sure if this is good way how to detect if Popup, not sure but this may break when user connects device on some different screen than homescreen?

Reply:
I think in general we don't necessarily need to redirect the user back to the home screen right?

Reply:
We can possible reuse hook proposed here https://github.com/trezor/trezor-suite/pull/14262#discussion_r1773210871